### PR TITLE
Adding QCD_HT_LO_MLM DataCards For Approval

### PR DESCRIPTION
**Pull request by rajat116**

### CODE REVIEW

**Comment by perrozzi** [comment]
the position of the cards must be
`bin/MadGraph5_aMCatNLO/cards/production`
and not
`bin/rgupta/cards/production`

File: bin/rgupta/cards/production/13TeV/QCD_HT1000toInf/QCD_HT1000toInf_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
```
Comment (perrozzi) [inline bin/rgupta/cards/production/13TeV/QCD_HT1000toInf/QCD_HT1000toInf_proc_card.dat]: please follow the cleanup instructions https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by rajat116** [comment]
@perrozzi  Should i put the card in bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/QCD?
or do i make some new folder in this directory to put my cards?

**Comment by perrozzi** [comment]
https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/QCD/QCD_HT_LO_MLM is good (check for already existing datacards to avoid duplicates)

**Comment by perrozzi** [comment]
hi, I had suggested a precise location, namely
`bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/QCD/QCD_HT_LO_MLM`
while you put them under
`bin/MadGraph5_aMCatNLO/cards/production/2017/QCD_HT_LO_MLM`
I am pretty sure that if you move the cards to the location I specificied you will find out that yours are duplicate. this means that somebody else is also going to produce these gridpacks. please check the spreadsheets with part1-2-3 requests and coordinate

**Comment by rajat116** [comment]
Hi @perrozzi , I have put in the wrong location by mistake. Now i have put them under specified location.
Also, there was only one file, which was same in that directory(HT50to100), so i didn't replaced that. I have just added 4 extra directories in that. 
Please let me know, if it is fine now?

**Comment by perrozzi** [comment]
Hi @rajat116, thanks. You might have realized that the directory names are indeed different, but just because you are using a different binning.....
Do you have a specific reason to replace the existing binning?

**Comment by rajat116** [comment]
Hi @perrozzi These specific binnings are asked by analyzers in our group. They need to compare it with 2015 low PU data.
I can ask him if he can use the existing binning. But I am not sure if these existing samples can be used to compare with old 2015 low PU data.

**Comment by perrozzi** [comment]
of they are needed to be compared to 2015 data, then they need to go to the pre2017 folder, not 2017
cfr https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Recent_modifications_to_the_genp

**Comment by rajat116** [comment]
I have moved the cards in the pre2017 folder:
bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/

Please have a look and let me know, if this is fine now.

Thanks
Rajat

**Comment by perrozzi** [comment]
hi, I am afraid you copied them instead of moving (i.e. I can still see them in the 2017 folder).
Another question: I strongly believe that QCD HT LO MLM samples exist for 2016 MC.
This means that you could reuse existing gridpacks.
The binning would likely be different from the one here, but still should cover the full phase space.
You could even reuse existing GEN-SIM if you are lucky...
Can I please ask you to check in DAS/McM about existing datasets an gridpacks?

**Comment by rajat116** [comment]
Hi @perrozzi , I found these samples on DAS:

/QCD_HT50to100_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM

/QCD_HTXXXtoXXX_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISummer16MiniAODv2-PUMoriond17(PUSpring16)_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM

XXXtoXXX : 100to200, 200to300, 300to500, 500to700, 700to1000, 1000to1500, 1500to2000, 2000toInf.

These samples are with PU (PUMoriond17 or PUSpring16). Can the gridpacks be used to make samples to compare with LowPU 2015 data.

Thanks
Rajat

**Comment by perrozzi** [comment]
> These samples are with PU (PUMoriond17 or PUSpring16). Can the gridpacks be used to make samples to compare with LowPU 2015 data.

yes indeed!

**Pull request closed without merging**